controllers/session_test.go

```diff
@@ -70,7 +70,7 @@ var _ = Describe("SessionController", func() {
 		})
 	})
 
-	Describe("POST /session/create", func() {
+	Describe("POST /user/session_create", func() {
```

Comment:
Why not simplying to `Describe("POST /user/session"`) 🤔 `POST` is already implicitly a create action.